_lessons/16-setting-up-the-layout.md

```diff
@@ -0,0 +1,161 @@
+  </body>
+</html>
+```
+
+The CSS for our project will go in-between the `style` tags in the `head`, and the HTML for everything displayed on the page will go in between the `body` tags. We also add a `title` tag to the `head` in order to display a page title in the browser tab. 
+
+Go ahead and save this in a new file with a `.html` extension. You can name the file anything you'd like, but the standard file name for a website's landing page is `index.html`. This name is historical, and refers to how the index is the first place you'd look if you were searching for something in a book. 
+
+### Code the Layout with Divs
+
+Before we start writing any code, we should first draw what we want our basic layout to look like. For this project, we will have a header, footer, sidebar and content. We'll place the header and footer at the top and bottom, and we'll place the sidebar and content side-by-side in between. The sidebar will go on the left, and content on the right. 
+
+<div style="text-align: center">
+  <img src="/assets/ch16_1.png" />
+</div>
+
+The way to translate this into code is to think of each section as its own `div`. To make it clear what each `div` is, we can give it a custom ID. So we can start our code by writing a `div` for each section: 
+
+```
+<div id="header">Header</div>
+<div id="sidebar">Sidebar</div>
+<div id="content">Content</div>
+<div id="footer">Footer</div>
+```
+
+Next, we think about what we need to do to arrange the divs. By default, divs are stacked one after another, vertically. However, we want the sidebar and content to be side-by-side. Remember that in the last section, we saw a way to place two divs side-by-side by putting them both inside a third div, and then setting some CSS (`display: flex;`) on the div they are inside of. 
+
+Let's do this now, and give this div a descriptive ID as well - how about "container"?
+
+```
+<div id="header">Header</div>
+<div id="container">
+  <div id="sidebar">Sidebar</div>
+  <div id="content">Content</div>
+</div>
+<div id="footer">Footer</div>
+```
+
+And we'll add a CSS rule adding the flexbox rule to the container: 
+
+```
+#container {
+  display: flex; 
+}
+```
+
+Now if we load the page, we see it looks a little more like what we were expecting. 
+
+<div class="screenshot">
+    <img src="/assets/ch16_2.png" />
+</div>
+
+### Set Custom Widths
+
+Let's add a little CSS to help us see what we are doing. We'll remove this later, but for now, let's put a black border around every `div` so that we can see how large it is on the page. (`#000000` is the hexadecimal color code for black.)
+
+```
+div {
+  border: 1px solid #000000; 
+}
+```
+
+Now, we can see that the header and footer are taking up the full width of the page. The sidebar and content are only as wide as the text inside of them. (That is a side effect of them being inside a <i>flex container</i>.) We can use the CSS `width` property to give them custom widths instead. 
```

Comment:
I would suggest adding a picture of the state of the world as of adding the border, prior to making any sizing adjustments. That way the whole "Now, we can see" phrase has something to actually point to.